ny-portal/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "ny-portal",
+  "version": "1.0.0",
+  "description": "Dokumentasjonsportal for Jøkul designsystem",
+  "license": "MIT",
+  "type": "module",
+  "scripts": {
+    "build": "cross-env NODE_OPTIONS=--no-deprecation next build",
+    "dev": "cross-env NODE_OPTIONS=--no-deprecation next dev",
+    "devsafe": "rm -rf .next && cross-env NODE_OPTIONS=--no-deprecation next dev",
+    "generate:importmap": "cross-env NODE_OPTIONS=--no-deprecation payload generate:importmap",
+    "generate:types": "cross-env NODE_OPTIONS=--no-deprecation payload generate:types",
+    "lint": "cross-env NODE_OPTIONS=--no-deprecation next lint",
+    "payload": "cross-env NODE_OPTIONS=--no-deprecation payload",
+    "start": "cross-env NODE_OPTIONS=--no-deprecation next start"
+  },
+  "dependencies": {
+    "@aws-sdk/client-sts": "^3.731.1",
+    "@aws-sdk/rds-signer": "^3.731.1",
+    "@fremtind/jokul": "^0.37.11",
+    "@payloadcms/db-postgres": "latest",
+    "@payloadcms/next": "latest",
+    "@payloadcms/richtext-lexical": "latest",
+    "clsx": "^2.1.1",
+    "cross-env": "^7.0.3",
+    "graphql": "^16.8.1",
+    "next": "15.1.0",
+    "payload": "latest",
+    "react": "19.0.0",
+    "react-dom": "19.0.0",
```

Er dette litt prematurt? Vi har vel enda ikke klart å jokul over på React 19.

---

Typene bør oppdateres også når vi negraderer til 18.

```suggestion
    "@types/react": "18.3.3",
    "@types/react-dom": "18.3.5",
```